custom_components/thermal_comfort/sensor.py

```diff
@@ -255,6 +267,38 @@ def computeRiskLevel(self, temperature, humidity):
             return 2  # Frost probable despite the temperature
         return 0  # No risk of frost
 
+    def computeSimmerIndex(self, temperature, humidity):
+        """https://www.vcalc.com/wiki/rklarsen/Summer+Simmer+Index"""
+        fahrenheit = util.temperature.celsius_to_fahrenheit(temperature)
+
+        si = (1.98 * (fahrenheit - (0.55 - (0.0055 * humidity)) * (fahrenheit - 58.0)) - 56.83)
+
+        if fahrenheit < 70:
+            si = fahrenheit
+        
```

Comment:
Theres some whitespace here when viewing the commit localy.

Reply:
![image](https://user-images.githubusercontent.com/1131371/148928326-5028e3c0-a97c-49bf-810c-084fbf1b9d1c.png)